crates/net/network/src/config.rs

```diff
@@ -132,14 +131,6 @@ impl<C> NetworkConfig<C> {
             builder = builder.fork(OPSTACK, fork_id)
         }
 
-        if chain == Chain::optimism_mainnet() || chain == Chain::base_mainnet() {
-            builder = builder.add_optimism_mainnet_boot_nodes()
-        } else if chain == Chain::from_named(NamedChain::OptimismSepolia) ||
-            chain == Chain::from_named(NamedChain::BaseSepolia)
-        {
-            builder = builder.add_optimism_sepolia_boot_nodes()
-        }
-
```

Comment:
how are you adding the boot nodes now that this is removed?

Reply:
By adding the nodes to the ChainSpec, NetworkConfig will have them to serve to discovery_v5_with_config_builder